### PR TITLE
Edit root readme

### PR DESCRIPTION
**Pull request by InquestGeronimo**
Removed the table in the resources section as it was interfering with the docs sync. Also did some grammar edits and added a new URL for the new `deepsparse.server` readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,16 +160,13 @@ Tasks Supported:
 - Question Answering
 - Masked Language Modeling (MLM)
```
Comment (mgoin) [inline README.md:161]: we could add the `task="blank"` strings here or link to the "Try It Now" docs for the listed tasks (talking about these ones https://neuralmagic.com/use-cases/#nlp)

**Review by mgoin — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (InquestGeronimo) [reply README.md:161]: good one

File: README.md [hunk 2]
```diff
@@ -154,20 +156,19 @@ NLP Tutorials:
...
 Tasks Supported: 
-- Text Classification (Sentiment Analysis)
-- Question Answering
-- Masked Language Modeling (MLM)
-
-__ __
+- [Named Entity Recognition](https://neuralmagic.com/use-cases/sparse-named-entity-recognition/)
+- [Multi-Class Classification](https://neuralmagic.com/use-cases/sparse-multi-class-text-classification/)
+- [Binary Text Classification](https://neuralmagic.com/use-cases/sparse-binary-text-classification/)
+- [Text Classification (Sentiment Analysis)](https://neuralmagic.com/use-cases/sparse-sentiment-analysis/)
```
Comment (jeanniefinks) [inline README.md:162]:
```suggestion
- [Text Classification: Sentiment Analysis](https://neuralmagic.com/use-cases/sparse-sentiment-analysis/)
```

File: README.md [hunk 3]
```diff
@@ -154,20 +156,19 @@ NLP Tutorials:
 - [Getting Started with Hugging Face Transformers 🤗](https://github.com/neuralmagic/deepsparse/tree/main/examples/huggingface-transformers)
 
 Tasks Supported: 
-- Text Classification (Sentiment Analysis)
-- Question Answering
-- Masked Language Modeling (MLM)
-
-__ __
+- [Named Entity Recognition](https://neuralmagic.com/use-cases/sparse-named-entity-recognition/)
+- [Multi-Class Classification](https://neuralmagic.com/use-cases/sparse-multi-class-text-classification/)
```
Comment (jeanniefinks) [inline README.md:160]:
```suggestion
- [Text Classification: Multi-Class](https://neuralmagic.com/use-cases/sparse-multi-class-text-classification/)
```

File: README.md [hunk 4]
```diff
@@ -154,20 +156,19 @@ NLP Tutorials:
 - [Getting Started with Hugging Face Transformers 🤗](https://github.com/neuralmagic/deepsparse/tree/main/examples/huggingface-transformers)
 
 Tasks Supported: 
-- Text Classification (Sentiment Analysis)
-- Question Answering
-- Masked Language Modeling (MLM)
-
-__ __
+- [Named Entity Recognition](https://neuralmagic.com/use-cases/sparse-named-entity-recognition/)
```
Comment (jeanniefinks) [inline README.md:159]:
```suggestion
- [Token Classification: Named Entity Recognition](https://neuralmagic.com/use-cases/sparse-named-entity-recognition/)
```

**Review by jeanniefinks — CHANGES_REQUESTED** [review]
Minor things!
Also, and this is totally my bad: may we on line 266 correct
`[**Deep Sparse Community Slack**](https://join.slack.com/t/discuss-neuralmagic/shared_invite/zt-q1a1cnvo-YBoICSIw3L1dmQpjBeDurQ)`
to
`**[Deep Sparse Community Slack**](https://join.slack.com/t/discuss-neuralmagic/shared_invite/zt-q1a1cnvo-YBoICSIw3L1dmQpjBeDurQ)**`
In the rendered html https://docs.neuralmagic.com/deepsparse/main/index.html - the bold markdown needs to go outside the link markdown. Confirm that will work when it does render again!

For other similar root README(s), we'll need to do the same. Thx!

File: README.md [hunk 5]
```diff
@@ -76,25 +76,25 @@ pip install deepsparse
 
 ## 🔌 DeepSparse Server
 
-The DeepSparse Server allows you to serve models and pipelines in deployment in CLI. The server runs on top of the popular FastAPI web framework and Uvicorn web server. Install the server using the following command:
+The DeepSparse Server allows you to serve models and pipelines from terminal. The server runs on top of the popular FastAPI web framework and Uvicorn web server. Install the server using the following command:
```
Comment (jeanniefinks) [inline README.md:79]:
```suggestion
The DeepSparse Server allows you to serve models and pipelines from the terminal. The server runs on top of the popular FastAPI web framework and Uvicorn web server. Install the server using the following command:
```

**Review by jeanniefinks — DISMISSED** [review]

File: README.md [hunk 6]
```diff
@@ -76,25 +76,25 @@ pip install deepsparse
...
-**⭐ Single Model ⭐**
+### Single Model
 
 Once installed, the following example CLI command is available for running inference with a single BERT model:
 
 ```bash
 deepsparse.server \
     --task question_answering \
-    --model_path "zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/base-none"
+    --model_path "zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned_quant-aggressive_95"
```
Comment (mgoin) [inline README.md:92]:
```suggestion
    --model_path "zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/12layer_pruned80_quant-none-vnni"
```

File: README.md [hunk 7]
```diff
@@ -124,27 +127,26 @@ deepsparse.benchmark [-h] [-b BATCH_SIZE] [-shapes INPUT_SHAPES]
...
-## 👩‍💻 NLP Inference | Question Answering
+
+## 👩‍💻 NLP Inference Example
 
 ```python
 from deepsparse.transformers import pipeline
 
 # SparseZoo model stub or path to ONNX file
-onnx_filepath="zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned-aggressive_98"
+model_path = "zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned-aggressive_98"
```
Comment (mgoin) [inline README.md:145]:
```suggestion
model_path = "zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/12layer_pruned80_quant-none-vnni"
```

**Review by mgoin — COMMENTED** [review]
just updating model stubs to latest QA models but looks good otherwise, will accept after

**Review by mgoin — APPROVED** [review]

**Review by jeanniefinks — APPROVED** [review]

**Pull request merged**